src/com/sun/ts/tests/jaspic/tssv/config/SOAPTSServerAuthConfig.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2007, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hmm, did you include a date range because you based this new file on com.sun.ts.tests.jaspic.tssv.config.TSServerAuthConfig (or similar file) that was from 2007?  I was first expecting that the Copyright would be 2020.

Reply:
I don't think this question blocks the review though, so will continue.  Please answer when you have time.